components/layout_2020/flexbox/layout.rs

```diff
@@ -863,6 +865,68 @@ impl FlexContainer {
         }
     }
 
+    /// Create a absolutely positioned flex child fragment, using the rules the
+    /// specification dictates. This should take into account the alignment and
+    /// justification values of the container and the child to position it within a
+    /// 'inset-modified containing block," which may be either the "static-position
+    /// rectangle" that's calculated below or a modified version of the absolute's
+    /// containing block adjusted by the insets specified in the item's style.
+    ///
+    /// From https://drafts.csswg.org/css-flexbox/#abspos-items:
+    /// > The cross-axis edges of the static-position rectangle of an
+    /// > absolutely-positioned child of a flex container are the content edges of the
+    /// > flex container The main-axis edges of the static-position rectangle are where
+    /// > the margin edges of the child would be positioned if it were the sole flex item
+    /// > in the flex container, assuming both the child and the flex container were
+    /// > fixed-size boxes of their used size. (For this purpose, auto margins are
+    /// > treated as zero.)
+    fn create_absolutely_positioned_flex_child_fragment(
+        &self,
+        absolutely_positioned_box: ArcRefCell<AbsolutelyPositionedBox>,
+        containing_block: &ContainingBlock,
+        container_size: FlexRelativeVec2<Au>,
+        positioning_context: &mut PositioningContext,
+    ) -> Fragment {
+        let alignment = {
+            let fragment = absolutely_positioned_box.borrow();
+            let make_flex_only_values_directional_for_absolutes =
+                |value: AlignFlags, reversed: bool| match (reversed, value.value()) {
```

Comment:
It's a bit confusing to match in the opposite order of the function parameters

```suggestion
                |value: AlignFlags, reversed: bool| match (value.value(), reversed) {
```

Reply:
I've switched these around.